samples/Sentry.Samples.AspNetCore.Grpc/Program.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Net;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Sentry.Samples.AspNetCore.Grpc
+{
+#pragma warning disable RCS1102
+    public class Program
```

Comment:
Any reason why it's not `static`?

Reply:
Not really, no. I'm not sure why I did that. I'll make it static.